_posts/2020-01-13-ols1-announcement.md

```diff
@@ -9,24 +9,7 @@ image: https://images.unsplash.com/photo-1533745848184-3db07256e163?ixlib=rb-1.2
 ---
 *We are excited to welcome this fantastic set of mentees who are not only located in different parts of the world, but also represent different identities and backgrounds, aim to address a wide range of questions in their field, and are motivated to bring a culture change in their areas. Many of them are long-standing Open Scientists who aim to use this opportunity to apply “Open by Design” principle in their projects through this program.*
 
-{% assign people = site.data.people %}
-{% assign projects = site.data.ols-1-projects %}
-{% assign experts-speakers = site.data.ols-1-experts-speakers %}
-
-{% assign all-participants = '' %}
-{% assign all-mentors = '' %}
-{% for project in projects %}
-    {% assign p-pparticipants = '' %}
-    {% for p in project.participants %}
-        {% capture all-participants %}{{ all-participants}}, [{{ people[p].first-name }} {{ people[p].last-name }}](/ols-1/projects-participants#{{ p }}){% endcapture %}
-    {% endfor %}
-    {% for m in project.mentors %}
-        {% capture all-mentors %}{{ all-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/ols-1#{{ m }}){% endcapture %}
-    {% endfor %}
-{% endfor %}
-
-{% assign p-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
-{% assign p-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
+{% include _includes/announcement-blog.html cohort='ols-1' %}
```

Comment:
The title could be `announcement-blog-metadata`

Reply:
Done. Thank you!